drivers/hmis_external_apis/app/jobs/hmis_external_apis/ac_hmis/fetch_clients_with_active_referrals_job.rb

```diff
@@ -6,6 +6,7 @@
 
 module HmisExternalApis::AcHmis
   class FetchClientsWithActiveReferralsJob < BaseJob
+    queue_as ENV.fetch('DJ_LONG_QUEUE_NAME', :long_running)
```

Comment:
Could maybe be short? I don't think it takes too long, fetches ids from an external API (with the order of 100s of responses, I think) and updates some records using batch operations

Reply:
We'd like to preference long, unless there's a need for the job to return quickly, or it's super light on resources, we have more capacity on the long queue.